tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraShortLivedTokenRoutesContract.scala

```diff
@@ -260,4 +261,88 @@ trait LinagoraShortLivedTokenRoutesContract {
            |}""".stripMargin)
   }
 
+  @Test
+  def getTokenShouldFailWithRevokedLongLivedToken(): Unit = {
+    val longLivedTokenRequest: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "create": {
+         |        "K38": {
+         |          "deviceId": "$DEVICE_ID"
+         |        }
+         |      }
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val longLivedTokenResponse: String = `given`()
+      .headers(getHeadersWith(BOB_BASIC_AUTH_HEADER))
+      .body(longLivedTokenRequest)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .extract()
+      .body()
+      .asString()
+
+    val longLivedTokenId: String = (((Json.parse(longLivedTokenResponse) \\ "methodResponses" )
+      .head \\ "created")
+      .head \\ "id")
+      .head.asInstanceOf[JsString].value
+
+    val longLivedToken: String = (((Json.parse(longLivedTokenResponse) \\ "methodResponses" )
+      .head \\ "created")
+      .head \\ "token")
+      .head.asInstanceOf[JsString].value
+
+    Thread.sleep(10000)
+
+    val revokeRequest: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["$longLivedTokenId"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    `given`
+      .headers(getHeadersWith(BOB_BASIC_AUTH_HEADER))
+      .body(revokeRequest)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+
+    Thread.sleep(10000)
+
+    val response: String = `given`()
+      .header(buildBearerTokenHeader(longLivedToken))
+      .queryParam("type", "shortLived")
+      .queryParam("deviceId", DEVICE_ID)
+    .when()
+      .basePath(BASE_PATH)
+      .get().prettyPeek()
+    .`then`()
+      .statusCode(SC_INTERNAL_SERVER_ERROR)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response)
+      .isEqualTo(
+        s"""{
+           |    "type": "about:blank",
+           |    "status": 500,
+           |    "detail": null
```

Comment:
I don't think it's correct either anyway, gonna need to change something in the code. should be 401 right?

Reply:
I updated here
https://github.com/linagora/tmail-backend/pull/249

Reply:
Was about to look at it, but thanks dearly appreciated :)